src/devices/develco.ts

```diff
@@ -684,10 +684,8 @@ const definitions: DefinitionWithExtend[] = [
         toZigbee: [develco.tz.led_control, develco.tz.ias_occupancy_timeout],
         exposes: (device, options) => {
             const dynExposes = [];
-            if (Number(device?.softwareBuildID?.split('.')[0]) >= 3) {
+            if (device && device.softwareBuildID && Number(device.softwareBuildID.split('.')[0]) >= 2) {
```

Comment:
```suggestion
            if (Number(device?.softwareBuildID?.split('.')[0]) >= 2) {
```

Does this also work?